README.md

```diff
@@ -1,5 +1,8 @@
 # Braintree iOS SDK
 
+## iOS v6 Beta
+Currently, version 6 of the iOS SDK is in Beta. For stable releases, please point to version 5 of the SDK. While preparing for general availability, we expect to make breaking changes in the beta releases.
```

Comment:
Nitpick - I think this might look a little weird above the badge icons + don't think we need a new section header.

Reply:
```
v5 is the latest major version of Braintree iOS. To update from v4, see the [v5 migration guide](https://github.com/braintree/braintree_ios/blob/master/V5_MIGRATION.md).
```

These #s and migration guide links should also be updated.

Reply:
Maybe that is where we can edit to include v6 beta info.